bower.json

```diff
@@ -8,6 +8,7 @@
         "Chart.js"
     ],
     "devDependencies": {
-        "jquery": "~2.1.4"
+        "jquery": "~2.1.4",
+        "moment": "2.10.6"
```

Comment:
This is okay.  I'm not sure who's contributing and using bower to install devDependencies, but it won't hurt.